packages/fungible-token/bridge-fungible-token/tests/harness.rs

```diff
@@ -266,6 +267,137 @@ mod success {
         assert_eq!(amount, config.overflow_2);
     }
 
+    #[tokio::test]
+    async fn claim_refund_of_wrong_token_deposit() {
+        // perform a failing deposit first to register a refund & verify it, then claim and verify output message is created as expected
```

Comment:
Can you make this comment a bit clearer?

Reply:
This is actually not mine - git diffs playing funny here. I can try elaborate the comment but since I 'm not the author I might end up introducing some mistakes

Reply:
Done